src/App.js

```diff
@@ -0,0 +1,83 @@
+import './App.css';
+import React, { useState , useRef } from 'react';
+import ChatMessageList from './chat/ChatMessageList'
+import { v4 as uuidv4 } from 'uuid';
+import axios from "axios";
+
+const BASE_URL = "http://localhost:8080"
+
+let postChatRequest = (message) => {
```

Comment:
Should add such function with its own logic/responsibilty in a different file, 

because the responsibility of this file is for structuring of the visual components